code/modules/research/designs/boards/machine_science.dm

```diff
@@ -103,22 +103,22 @@
 	id = "accelerator"
 	build_path = /obj/item/weapon/circuitboard/anom/accelerator
 
-/datum/design/anom/accelerator
```

Comment:
I don't think anyone ever used them once. Doesn't look like anyone ever reported it.